src/components/MonthlyGood/MonthlyGoodSelectorWrapper.vue

```diff
@@ -74,12 +74,16 @@ export default {
 			}
 		},
 		initStickyBehavior() {
-			// Initial position down the page of element before mg selector element
-			// const { bottom } = this.$el.getElementsByClassName('monthly-good-info')[0].getBoundingClientRect();
+			// temporary set to not sticky to calculate these values based on original element position on page
+			this.isSticky = false;
+			this.mgStickBarOffset = 0;
+
+			// Initial position down the page of selector
+			const { top } = this.$el.getElementsByClassName('monthly-good-selector')[0].getBoundingClientRect();
```

Comment:
This used to be based on the positioning of the previous element. Made this so it doesnt depend on other elements. It's a bit tricky because on resize the sticky calculations should be based on where the non-sticky version of the element is on the page, not where it is currently (since it could be sticky when the window is resized)